pipelines/apDetectorVisitQualityCore.yaml

```diff
@@ -12,3 +12,11 @@ tasks:
       atools.simpleSky: SimpleDiaPlot
       python: |
         from lsst.analysis.tools.atools import *
+  analyzeTrailedDiaSrcCore:
+    class: lsst.analysis.tools.tasks.AssocDiaSrcDetectorVisitAnalysisTask
+    config:
+      connections.data: goodSeeingDiff_longTrailedSrc
```

Comment:
Since this metric will only work in the situation where this exact data product is written out, I'm not convinced it belongs in the "core" pipeline. What do you think the ideal behavior is for when this data product doesn't exist? Or maybe you're just expecting users to be explicit about which tasks they want from this pipeline, i.e., specify just `analyzeAssocDiaSrcCore` if you don't want the rest?

Second question, can you please be more generic about the data product's name, i.e., allow for `goodSeeing` or `deep` vs hard-wiring `goodSeeing`?

Reply:
I am further confused about why the class for this task is the same as the class for the task above it... I think the place to specify the connections is in e.g. `assocDiaSrcDetectorVisitAnalysis.py`, or a new task analogous to that one that you write, not here.

Reply:
Having the same task was recommended by Claire as there was nothing particularly unique about the task since it was just repeating similar results to diaSrc. However, that didn't allow for the generic names, so I ended up writing out the separate task anyways just to make it more universal.